src/geometry/ramer_douglas_peucker.rs

```diff
@@ -0,0 +1,73 @@
+use crate::geometry::Point;
+
+pub fn ramer_douglas_peucker(polygon: &[Point], epsilon: f64) -> Vec<Point> {
```

Comment:
Do you really mean `polygon`? I thought this should work for any _polygonal chain_.

Reply:
Yes it does, you are right, should i name it `points`?